contents/handbook/people/team-structure/team-structure.md

```diff
@@ -51,3 +53,24 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 ## [People & culture](people)
 
 - [Eltje Lange](/handbook/people/team#eltje-lange-people-and-talent) (People and Talent)
+
+
+# Organization changes
+
+We’re still an early stage company, and so is our product. We strongly value moving and shipping fast (see core values) and we constantly iterate not only our product but our organization too. As this happens, it may be quite possible that organizational changes occur. This mostly happens on the small teams plane. To make the changes smoother for everyone, here’s the checklist we use:
```

Comment:
```suggestion
We’re still an early stage company, and so is our product. We strongly value moving and shipping fast (see [core values](/handbook/company/values)) and we constantly iterate not only our product but our organization too. As this happens, it may be quite possible that organizational changes occur. This mostly happens on the [small teams](/handbook/people/team-structure/why-small-teams) plane. To make the changes smoother for everyone, here’s the checklist we use:
```